tests/flows/test_vesting.py

```diff
@@ -72,11 +51,29 @@ def test_main(
 
     permissions = erc721_nft.get_permissions(consumer_wallet.address)
 
-    assert permissions[1] is True
-    assert permissions[2] is True
-    assert permissions[3] is True
+    assert permissions[ERC721Permissions.MANAGER] is False
+    assert permissions[ERC721Permissions.DEPLOY_ERC20] is True
+    assert permissions[ERC721Permissions.UPDATE_METADATA] is True
+    assert permissions[ERC721Permissions.STORE] is True
```

Comment:
Remove "is True", "is False" like above

Reply:
I removed all the redundant statements like this for `test_vesting.py` file.